packages/core/src/routes.js

```diff
@@ -11,7 +11,7 @@ type Routes = {
 }
 
 const routes: Routes = {
-    '/': { exact: true, component: () => 'Geodashboard Core' }
+    '/': { exact: true, component: Tests }
```

Comment:
Having the `Tests` component at root breaks the URLs for the test dropdown menu that are set in `BaseTests` component. We can make a ticket for this and fix it later after this one is merged.

Reply:
Should I revert it then?

Reply:
It's not a big deal. You can either revert it or just create a ticket for it and we can fix it later.